portal/package-lock.json

```diff
@@ -1,12563 +1,8 @@
 {
   "name": "dtfs2",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
this needs to be `2` - you may need to update npm to the latest version

Reply:
Updated 👍